package/Dockerfile

```diff
@@ -1,8 +1,21 @@
-FROM ghcr.io/oracle/oraclelinux:7-slim
+FROM ghcr.io/verrazzano/golang:v1.20.12 as build
+
 RUN yum update -y \
     && yum clean all \
     && rm -rf /var/cache/yum/*
-COPY bin/backup-restore-operator /usr/bin/
+
+COPY . /go/src/github.com/backup-restore-operator/
+RUN  cd /go/src/github.com/backup-restore-operator/scripts && \
+    ./build
+
+FROM ghcr.io/verrazzano/golang:v1.20.12
```

Comment:
This from needs to be ghcr.io/oracle/oraclelinux:8-slim

Reply:
You want to add these two Iines after line 11

RUN microdnf -y update && \
    microdnf -y clean all

Reply:
Thanks, will do the changes.